src/Tool-Base/PharoShortcuts.class.st

```diff
@@ -249,7 +249,7 @@ PharoShortcuts >> forwardLineShortcut [
 
 { #category : #'keymaps - editor navigation' }
 PharoShortcuts >> forwardWordShortcut [
-	^ Character arrowRight meta
+	^ Character arrowRight ctrl
```

Comment:
I'm not sure that this change will work on Mac.

Reply:
Changed + Added comment